src/fix_gemc.cpp

```diff
@@ -0,0 +1,514 @@
+// clang-format off
+/* ----------------------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/, Sandia National Laboratories
+   LAMMPS development team: developers@lammps.org
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+// Gen Questions:
+// 1) What is a tagint?
+// 2) What are all the nullptrs in the definition of a fix
+
+/* ----------------------------------------------------------------------
+   Contributing author: Paul Crozier, Aidan Thompson (SNL)
```

Comment:
The author contribution line needs to be updated.

Reply:
Fixed.